src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/image/AzureImageOperation.java

```diff
@@ -43,7 +44,7 @@ public AzureImageOperation(String region) {
                         String offerName = offer.name();
                         String skuName = sku.name();
                         ImageSummary imageSummary = AzureImageOperationUtil.buildImageSummaryBy(publisherName, offerName, skuName);
-                        String id = UUID.randomUUID().toString();
+                        String id = AzureImageOperationUtil.encodeImageId(imageSummary.getId());
```

Comment:
If this line throws an exception, the whole method will fail. Is it what we want?

Reply:
No. It's fixed now.